LoggingKata/Program.cs

```diff
@@ -15,15 +15,44 @@ static void Main(string[] args)
             logger.LogInfo("Log initialized");
 
             var lines = File.ReadAllLines(csvPath);
+            var parser = new TacoParser();
+            var locations = lines.Select(parser.Parse).ToList();
 
-            logger.LogInfo($"Lines: {lines[0]}");
+            // Find the two Taco Bells in Alabama that are the furthest from one another.
+            
+            ITrackable tbell1 = locations[0];
+            ITrackable tbell2 = locations[0];
+            double distance = 0;
+            logger.LogInfo("Searching locations...");
 
-            var parser = new TacoParser();
+            for (int i = 0; i < locations.Count(); i++)
+            {
+                var locA = locations[i].Location;
+                var coordA = new GeoCoordinate(locA.Latitude, locA.Longitude);
+                
+                for (int j = 0; j < locations.Count(); j++)
+                {
+                    var locB = locations[j].Location;
+                    var coordB = new GeoCoordinate(locB.Latitude, locB.Longitude);
 
-            var locations = lines.Select(parser.Parse);
+                    var dist = coordA.GetDistanceTo(coordB);
+                    if (dist > distance)
+                    {
+                        distance = dist;
+                        tbell1 = locations[i];
+                        tbell2 = locations[j];
+                    }
+                }
+            }
 
-            // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            Console.WriteLine("\nLOCATION 1");
+            Console.WriteLine("  Name: " + tbell1.Name);
+            Console.WriteLine("  Point: " + tbell1.Location.ToString());
+            Console.WriteLine("\nLOCATION 2");
+            Console.WriteLine("  Name: " + tbell2.Name);
+            Console.WriteLine("  Point: " + tbell2.Location.ToString());
+            
```

Comment:
Small Bonus Challenge: Can you display the max distance formatted in a user-friendly way?